src/pages/user_profile/AffiliationAndRoles.jsx

```diff
@@ -197,62 +46,16 @@ export default function AffiliationAndRole(props) {
       Affiliation & Role
     </h1>
     <div style={{ marginTop: '20px' }} />
-    <div
-      style={{
-        color: '#000',
-        fontFamily: 'Montserrat',
-        fontSize: '16px',
-        fontStyle: 'normal',
-        fontWeight: '600',
-        lineHeight: 'normal'
-      }}
-    >
-      <p>Institution</p>
-      {!(profile.institutionId) && <div className={'alert-danger alert'} style={{fontWeight: 'normal'}}>
-        Please select an existing Institution. If you don&apos;t see your institution listed, please select the Contact
-        Us link to request that your institution be added.
-      </div>}
+    <div>
+      <p style={subHeadStyle}>Institution</p>
+      <div style={{ marginTop: '15px' }} />
+      {institution
+        ? <div data-cy='institutional-affiliation'>{institution?.name}</div>
```

Comment:
I believe the `?` isn't necessary as it's inside a block where `institution` is known to be defined.
```suggestion
        ? <div data-cy='institutional-affiliation'>{institution.name}</div>
```

Reply:
Good point, I think this was a leftover from an earlier version I had in place.